DataManagementSystem/scripts/dirac-admin-user-quota.py

```diff
@@ -2,17 +2,26 @@
 ########################################################################
 # $HeadURL$
 ########################################################################
-__RCSID__ = "$Id$"
-import DIRAC
-from DIRAC.Core.Base import Script
-
-Script.setUsageMessage("""
+"""
 Show storage quotas for specified users or for all registered users if nobody is specified
 
 Usage:
-   %s [user1 ...]
-""" % Script.scriptName)
+   dirac-admin-user-quota [user1 ...]
+Example:
```

Comment:
I think you need a line break before and after `Example:`. @andresailer should confirm

Reply:
It won't hurt, but it could be that the help message parser adds one before by itself.

Reply:
https://github.com/DIRACGrid/DIRAC/blob/e4a1302fb5ff7b3cd21e2eac28e8b75a19d6c918/docs/diracdoctools/cmd/commandReference.py#L205